hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/OFSPath.java

```diff
@@ -263,6 +264,23 @@ public boolean isBucket() {
         !this.getVolumeName().isEmpty();
   }
 
+  /**
+   * If volume and bucket names are not empty and the key name
+   * only contains the snapshot indicator, then return true.
+   * e.g. /vol/bucket/.snapshot is a snapshot prefix.
+   */
+  public boolean isSnapshotPrefix() {
```

Comment:
we have used the term "prefix" in so many different contexts it is a bit confusing.  What do you think about calling this method "isSnapshotIndicator"  to be consistent with the OM_SNAPSHOT_INDICATOR constant used below?